vocode/streaming/streaming_conversation.py

```diff
@@ -331,9 +213,45 @@ def __init__(
                 * TEXT_TO_SPEECH_CHUNK_SIZE_SECONDS
             )
 
-        async def process(self, item: InterruptibleEvent[BaseMessage]):
+        def send_filler_audio(self):
+            assert self.conversation.filler_audio_worker is not None
+            self.conversation.logger.debug("Sending filler audio")
+            if self.conversation.synthesizer.filler_audios:
+                filler_audio = random.choice(
+                    self.conversation.synthesizer.filler_audios
+                )
+                self.conversation.logger.debug(f"Chose {filler_audio.message.text}")
+                event = self.interruptible_event_factory.create(
+                    filler_audio,
+                    is_interruptible=filler_audio.is_interruptible,
+                )
+                self.conversation.filler_audio_worker.consume_nonblocking(event)
+            else:
+                self.conversation.logger.debug(
+                    "No filler audio available for synthesizer"
+                )
```

Comment:
Shouldn't the filler audio play prior to the agent response?

Reply:
the agent can decide to "send filler audio" (`AgentResponseFillerAudio`) which will actually get invoked via the AgentResponseWorker